growthtools/ad-spend-calc
# Basic layout

Fixes [this To Do](https://3.basecamp.com/3097764/buckets/19037960/todos/3078010187)

### Testing notes

1. Open the page.
1. See everything styled up except for inside the main content card.

![image](https://user-images.githubusercontent.com/75132/94990782-b65af600-054c-11eb-864d-2075cf0c40b5.png)


### Other information
